web/src/js/components/Header/FileMenu.jsx

```diff
@@ -50,5 +57,6 @@ export default connect(
         clearFlows: flowsActions.clear,
         loadFlows: flowsActions.upload,
         saveFlows: flowsActions.download,
+        openModal: modalActions.setActiveModal,
```

Comment:
We can make this `openOptions: () => modalActions.setActiveModal('OptionModal')`. I'm surprised we don't need to call `e.preventDefault()?` as with the other options?